docs/learn/learn-async-backing.md

```diff
@@ -0,0 +1,146 @@
+---
+id: learn-async
+title: Asynchronous Backing
+sidebar_label: Asynchronous Backing
+description: A brief overview of asynchronous backing, and how it affects Polkadot's scalability.
+keywords: [parachains, slots, backing, parablock]
+slug: ../learn-async
+---
+
+Asynchronous backing is a mechanism that introduces an enhancement to the block validation pipeline
+for parachain **<->** relay chain communication. This pipeline will allow collators to include more
```

Comment:
Is there a source for the factor of 5-10x?

Reply:
This is Rob's best "scratch paper" estimate. Though he recently directed me to scale back the estimate so as to not set expectations too high. We should use 3-5x rather than 5-10x here.

Sophia is getting blockspace benchmarking working, such that we'll have real before vs after data soon.